CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+<a name="1.1.5"></a>
```

Comment:
There's a way to manage this file automatically using semvers.  Check out:

https://github.com/semantic-release/semantic-release (used by https://github.com/godaddy/terminus)

Integrating with either of those is out of scope of this PR, but something to consider.

Reply:
This looks great! I just started looking into travis CI so this fits perfectly!